### PR TITLE
CORE-868: updated CyVerseAppBar.stories.js to work with the user prof…

### PR DESCRIPTION
**Pull request by slr71**
…ile provider

I forgot about Storybook when I added the `UserProfileProvider` component. 😖

### CODE REVIEW

File: stories/CyVerseAppBar.stories.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
 import React, { Component } from "react";
 import { storiesOf } from "@storybook/react";
 import CyverseAppBar from "../src/components/appBar/CyVerseAppBar";
+import { UserProfileProvider } from "../src/contexts/userProfile";
+import fetchMock from "fetch-mock";
+
+const mockUser = {
+    id: "mockUser",
```
Comment (psarando) [inline stories/CyVerseAppBar.stories.js]: Your mother was a 🐹 and your father smelled of 🍓

Comment (slr71) [reply stories/CyVerseAppBar.stories.js]: 🤣

Comment (slr71) [reply stories/CyVerseAppBar.stories.js]: Now I'm waiting to see when we pull the farm-animal catapults out. 😆

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**